webdriver-spec.html

```diff
@@ -3237,183 +3237,384 @@ <h2 id=element-displayedness>Element Displayedness</h2>
       What happens to unhandled JS errors? Caused by a user's JS? Caused by JS
       on a page? How does a user of the API obtain the list of errors? Is that
       list cleared upon read?
-    </div>
-    <p class="note">Conformance tests for this section can be found in the <a href="https://github.com/w3c/web-platform-tests/tree/master/webdriver/">webdriver module</a> under the "ecmascript" folder.</p>
+  </div>
 
-    <section>
-      <h2>Javascript Command Parameters</h2>
+  <p>When required to <dfn>JSON deserialize</dfn> with
+  argument <var>input value</var>, and optional
+  argument <var>seen</var>, which is always ignored, a remote end must
```

Comment:
My point was that in this case we never add a value to it; it's just there to make reusing the `clone an object` algorithm simpler.